IOS/SwiftSeedProject/SwiftSeedProject/AppDelegate.swift

```diff
@@ -18,26 +19,43 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
 
-    func applicationWillResignActive(_ application: UIApplication) {
-        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-        // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
-    }
-
-    func applicationDidEnterBackground(_ application: UIApplication) {
-        // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
-        // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
-    }
-
-    func applicationWillEnterForeground(_ application: UIApplication) {
-        // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
-    }
-
-    func applicationDidBecomeActive(_ application: UIApplication) {
-        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
-    }
-
-    func applicationWillTerminate(_ application: UIApplication) {
-        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    func applicationWillResignActive(_ application: UIApplication) { }
+
+    func applicationDidEnterBackground(_ application: UIApplication) { }
+
+    func applicationWillEnterForeground(_ application: UIApplication) { }
+
+    func applicationDidBecomeActive(_ application: UIApplication) { }
+
+    func applicationWillTerminate(_ application: UIApplication) { }
+    
+    // MARK: - Core Data stack
+    //TODO: Evaluate if having Core Data's stack in AppDelegate is the best approach.
+    lazy var persistentContainer: NSPersistentContainer = {
+        let container = NSPersistentContainer(name: "SwiftSeedModel")
+        container.loadPersistentStores(completionHandler: { (storeDescription, error) in
+            if let error = error as NSError? {
+                // TODO: Proper Error handling should be done here.
+                // fatalError() causes the application to generate a crash log and terminate. While useful in development this should be changed later. - Mugu.
+                fatalError("Unresolved error \(error), \(error.userInfo)")
+            }
+        })
+        
+        return container
+    }()
+    
+    // MARK: - Core Data Saving support
+    func saveContext () {
```

Comment:
Same here, default accessibility modifier is not restrictive enough for a method like `saveContext`